Edits to incorporate HEDIS criteria

mryanNCQA opened the pull request:
Various edits to accommodate HEDIS criteria.

Please feel free to accept/reject changes as you see fit.

mryanNCQA committed:
Edits to incorporate HEDIS criteria

brynrhodes reviewed (changes requested):
Thanks so much for this! The only question I have is whether we can change from using codes to using value sets? One of the things I'm trying to demonstrate with this is how we can use concepts within CQL to support mapping to different terminologies. So instead of specifying a particular code, defining the concept within CQL using a value set reference, and then (by keeping all the concept references within a concepts library) being able to swap out mappings with different versions of that library.

mryanNCQA committed:
Removing DRCs

mryanNCQA commented:
I've removed the references to single codes and updated the logic to reference the original value sets.

The three codes I had pulled out are already in the corresponding value sets, so calling the value set should work just fine.
